### PR TITLE
Improve Concepts section

### PR DESCRIPTION
**Pull request by lex111**
Signed-off-by: Alexey Pyltsyn <lex61rus@gmail.com>

### CODE REVIEW

**Comment by lex111** [comment]
Signed CLA.

**Comment by lex111** [comment]
Any news?

**Comment by zparnold** [comment]
@lex111 Can you fix the merge conflicts?

**Comment by lex111** [comment]
@zparnold done :heavy_check_mark:

File: content/en/docs/concepts/containers/container-lifecycle-hooks.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Here is some example output of events from running this command:
 
 ```
 Events:
-  FirstSeen  LastSeen  Count  From                                                   SubobjectPath          Type      Reason               Message
+  FirstSeen  LastSeen  Count  From                                                   SubObjectPath          Type      Reason               Message
```
Comment (sftim) [inline content/en/docs/concepts/containers/container-lifecycle-hooks.md]: Is this right?

Comment (lex111) [reply content/en/docs/concepts/containers/container-lifecycle-hooks.md]: Yes, that's correct.

**Pull request merged**